Libplanet/Net/Swarm.cs

```diff
@@ -1276,18 +1284,12 @@ CancellationToken cancellationToken
             }
             catch (IOException e)
             {
-                _logger.Error(
-                    e,
-                    $"IOException occured in DialAsync ({peer.EndPoint}).");
-                dealer.Dispose();
+                throw e;
             }
```

Comment:
This `catch` condition seems able to be safely removed.

Reply:
It's mistake. (`dealer.Dispose()` should be kept.) I'll fix it.